tests/replication/DealReplicationService.spec.ts

```diff
@@ -89,17 +89,6 @@ describe('DealReplicationService', () => {
         message: ErrorMessage[ErrorCode.REPLICATION_NOT_FOUND],
       })
     });
-    it('should return error if the replication request has already completed', async () => {
```

Comment:
Why is this test removed. 
Also, you need to add a test to test if a deal has already been proposed previously, it will be proposed again if `force` is set to true

Reply:
Because I wanted to be able to resume/retry a completed / error replication request - so that the deals can be resent when the SP corrects their error, for example min size too big.

Reply:
I see.